### PR TITLE
Add 3639 to service portfolio

### PR DESCRIPTION
**Pull request by rathoreraju**
Add 3639 to service portfolio

Note - Generated json via mybring-service-country-vas-common ServiceVasMappingJsonGenerator add change for 3639

### CODE REVIEW

**Comment by WebDevBring** [comment]
[trello.com/c/o6DRNCrs](https://trello.com/c/o6DRNCrs)

**Review by PraveenPandey01 — COMMENTED** [review]
@rathoreraju Did you generate this json after making changes in common-vas-lib and running `ServiceVasMappingJsonGenerator`? 
image

**Comment by rathoreraju** [comment]
> @rathoreraju Did you generate this json after making changes in common-vas-lib and running `ServiceVasMappingJsonGenerator`? image image

If I don't want direct update in dev-site and want to go through PR review and merge in that case till need above process?

**Comment by PraveenPandey01** [comment]
> > @rathoreraju Did you generate this json after making changes in common-vas-lib and running `ServiceVasMappingJsonGenerator`? image image
> 
> If I don't want direct update in dev-site and want to go through PR review and merge in that case still need above process?

yes, otherwise someone else's change may re-write your update from dev-site. I usually update in common-vas-lib first, generate required json for dev-site and merge 2 PRs together (common-vas and dev-site). Is it like we are updating in dev-site but aren't actually supporting the change??

File: data/services_vas.json
Comment (PraveenPandey01) [inline data/services_vas.json:1]: Are we sure that 1000 (Cash on delivery) is supported by 3639? I think not

image

Currently on dev-site prod
image

File: data/services_bsg.json
Comment (PraveenPandey01) [inline data/services_bsg.json:1]: Is it possible to show countries differently?? scroll seems to be too long 

image

something like this maybe 
image
We can check with Kateryna also as to how it should look

Comment (PraveenPandey01) [reply data/services_bsg.json:1]: @rathoreraju you could check look n feel in the staged environment
image

Comment (rathoreraju) [reply data/services_bsg.json:1]: updated UI now showing like ALL_COUNTRIES EXCEPT(NO, AF, BT, IL, LY, SS, SD, SY, YE)

**Review by PraveenPandey01 — APPROVED** [review]
LGTM

**Pull request merged**